debug_scripts/send_validator_logs.py

```diff
@@ -0,0 +1,48 @@
+#!/bin/python3
+import argparse
```

Comment:
I'd also:
- split this code into 2 functions (one to extract data, the other to send)
- add the test for the extracting function part - where it reads a small 'example' log file with a couple of lines and we check that it filters them correctly

Reply:
Split the code in two functions and added a test as requested.